### PR TITLE
Portfolio sections complete

### PR DESCRIPTION
**Pull request by tinatsina**
added portfolio section to project
update style.css
update button hover
update index.html
update removed all linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,5 +50,213 @@ <h1 class="developer-name">
...
+                <img src="img/08.png" alt="">
+            </div>
+            <div class="project-shaper">
+                <div class="project-details">
+                    <h1 class="item1 project-name-here">
+                        Project name goes here
+                    </h1>
+                    <div class="item2 built-with">HTML/CSS</div>
+                    <div class="item3 built-with">Ruby on Rails</div>
+                    <div class="item4 built-with">JavaScript</div>
```
Comment (mukhammadrizooff) [inline index.html:132]: - [ ] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list of `ul` and each of them a `li`.

File: index.html [hunk 2]
```diff
@@ -50,5 +50,213 @@ <h1 class="developer-name">
...
+                <img src="img/09.png" alt="">
+            </div>
+            <div class="project-shaper">
+                <div class="project-details">
+                    <h1 class="item1 project-name-here">
+                        Project name goes here
+                    </h1>
+                    <div class="item2 built-with">HTML/CSS</div>
+                    <div class="item3 built-with">Ruby on Rails</div>
+                    <div class="item4 built-with">JavaScript</div>
```
Comment (mukhammadrizooff) [inline index.html:153]: - [ ] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list of `ul` and each of them a `li`.

File: index.html [hunk 3]
```diff
@@ -50,5 +50,213 @@ <h1 class="developer-name">
...
+                <img src="img/10.png" alt="">
+            </div>
+            <div class="project-shaper">
+                <div class="project-details">
+                    <h1 class="item1 project-name-here">
+                        Project name goes here
+                    </h1>
+                    <div class="item2 built-with">HTML/CSS</div>
+                    <div class="item3 built-with">Ruby on Rails</div>
+                    <div class="item4 built-with">JavaScript</div>
```
Comment (mukhammadrizooff) [inline index.html:174]: - [ ] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list of `ul` and each of them a `li`.

File: style/style.css [hunk 4]
```diff
@@ -90,3 +90,246 @@ li {
...
   width: 100%;
 }
+
+.portfolio-grid-container {
+  display: grid;
+}
+
+.grid-flex {
+  display: block;
+}
```
Comment (mukhammadrizooff) [inline style/style.css:100]: - [ ] [OPTIONAL] Good job so far! Your project opens only the same size in the mobile version, resulting in a white field on the right when expanded from 300 px to 800 px. My advice to you would be to ask you to fix this problem. Let's learn [additional information](https://youtu.be/yU7jJ3NbPdA).

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @tinatsina, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights
- All linters passed 💯 
- Correct use of Gitflow 💯 
- Descriptive messages 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -50,5 +50,213 @@ <h1 class="developer-name">
...
+                <img src="img/07.png" alt="">
+            </div>
+            <div class="project-shaper">
+                <div class="project-details">
+                    <h1 class="item1 project-name-here">
+                        Project name goes here
+                    </h1>
+                    <div class="item2 built-with">HTML/CSS</div>
+                    <div class="item3 built-with">Ruby on Rails</div>
+                    <div class="item4 built-with">JavaScript</div>
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply index.html:69]: resolved the issue and replaced all the `div` with `ul` and all elements as respective `li`

**Review by jaflih — APPROVED** [review]
Hi @tinatsina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
All linters passed 💯
Correct use of Gitflow 💯
Descriptive messages 💯

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**